src/odb/include/odb/db.h

```diff
@@ -9223,6 +9271,116 @@ class dbAccessPoint : public dbObject
   // User Code End dbAccessPoint
 };
 
+class dbPowerDomain : public dbObject
+{
+ public:
+  // User Code Begin dbPowerDomainEnums
+  // User Code End dbPowerDomainEnums
+  const char* getName() const;
+
+  void setPowerSwitch(dbPowerSwitch* power_switch);
+
+  dbPowerSwitch* getPowerSwitch() const;
+
+  void setIsolation(dbIsolation* isolation);
+
+  dbIsolation* getIsolation() const;
+
+  // User Code Begin dbPowerDomain
+  static dbPowerDomain* create(dbBlock* block,
+                               const char* name);
+  static void destroy(dbPowerDomain* pd);
+
+  void addElement(const char* element);
+  std::vector<std::string> getElements();
```

Comment:
Shouldn't this return the list of matching elements?

Reply:
I'm not sure what you mean here?  Do you mean the associated dbInst?  I think that will come in the future as we flesh out the usage of UPF.